bin/compile

```diff
@@ -19,28 +19,28 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 
 BUILDPACK_DIR="$(dirname $(dirname $0))"
+INSTALL_DIR="/app/.awscli"
 
 echo "-----> Installing AWS CLI"
 curl --silent --show-error -o /tmp/awscli-bundle.zip "$AWS_CLI_URL" |& indent
-unzip -qq -d "$BUILD_DIR/vendor" /tmp/awscli-bundle.zip |& indent
+unzip -qq -d "$BUILD_DIR/.awscli" /tmp/awscli-bundle.zip |& indent
 
 # Since build-dir isn't guaranteed to be /app, move whatever's in /app's awscli
 # dir out of the way. The installer uses virtualenv, which means it must be
 # installed in the final location - we have to install into /app.
 rm -rf /tmp/awscli
-[ -e /app/vendor/awscli ] && mv /app/vendor/awscli /tmp/awscli
+[ -e /app/.awscli ] && mv /app/.awscli /tmp/awscli
```

Comment:
I know this is pre-existing, but this usage as well as the others below could instead use `${INSTALL_DIR}` too

Reply:
Good idea. Done.